### PR TITLE
Add support to bootstrap 0.14

### PR DESCRIPTION
**Pull request by brenovieira**
- Added support to bootstrap 0.14.X.
- Added test pages for supported angular/bootstrap versions

#28.

### CODE REVIEW

File: src/gm.datepickerMultiSelect.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ SOFTWARE.
 
 					/* Fires when date is selected or when month is changed. */
 					scope.$watch(function () {
-						return ctrl.activeDate.getTime();
+						return ctrl.activeDate ? ctrl.activeDate.getTime() : false;
```
Comment (spongessuck) [inline src/gm.datepickerMultiSelect.js]: Why this change? You don't want to watch 'false,' and it will silently fail if ctrl.activeDate isn't defined.

Comment (brenovieira) [inline src/gm.datepickerMultiSelect.js]: If you don't set a proper activeDate (like this.activeDate = new Date()) at
controller, it throws an exception, because it is null or undefined.

So I added this just to not raise an exception.

I think returning false doesn't create another $watch.

Let me know if you want a plunker of this or if you want to I revert this.
Em 13 de out de 2015 13:33, "spongessuck" <notifications@github.com>
escreveu:

Comment (spongessuck) [inline src/gm.datepickerMultiSelect.js]: Oh really? OK I think it's fine then.

If you would, I'd like to see it in a plunker before I merge.

**Comment by brenovieira** [comment]
@spongessuck ,

As I commented on my new commit,
Here is the history of [datepicker.js on bootstrap github](https://github.com/angular-ui/bootstrap/commits/master/src/datepicker/datepicker.js).

One of the lastest commits, they added three controllers to `daypicker` directive, as you can see here: [Before](https://github.com/angular-ui/bootstrap/commit/44354f67e55c571df28b09e26a314a845a3b7397?diff=split#diff-6240fc17e068eaeef7095937a1d63eaeL251) and [After](https://github.com/angular-ui/bootstrap/commit/44354f67e55c571df28b09e26a314a845a3b7397?diff=split#diff-6240fc17e068eaeef7095937a1d63eaeR462)

So, `ctrl` on
```
scope.$watch(function () {
	return ctrl.activeDate.getTime();
}, update);
```
is now actually an array of three controllers.
Then `ctrl.activeDate` is undefined and `ctrl.activeDate.getTime()` throws an exception.

I needed to change it to
```
scope.$watch(function () {
	return angular.isArray(ctrls)
		? ctrls[0].activeDate.getTime()
		: ctrls.activeDate.getTime();
}, update);
```

Note: I moved the test pages from root to test folder and forgot to change the path of `modules\app.js` and `dist\gm.datepickerMultiSelect.min.js`.

I forked your plunker and just change it to use bootstrap 0.14.1.
You can see the exception on console: http://plnkr.co/edit/EzHmqn?p=preview

**Comment by spongessuck** [comment]
Sure, I think it looks great. Better too much info than not enough and it's not obvious what's going on.
